packages/api/src/add-build.js

```diff
@@ -46,7 +47,7 @@ async function addBuild (testCases, buildInfo, client, collectionName = 'reposit
   // if this build has already been posted, skip
   const thisBuildExists = await dbRepo.collection('builds').doc(buildInfo.buildId).get();
   if (thisBuildExists.exists) {
-    return;
+    buildInfo.buildId = buildInfo.buildId.toString() + uuidv4();
```

Comment:
will this show up in the UX in places where we show the workflow ID?

<img width="305" alt="Screen Shot 2020-07-24 at 5 07 44 PM" src="https://user-images.githubusercontent.com/194609/88444092-36881100-cdd0-11ea-98ec-c545d89e3ab6.png">

What if we made all `buildId`s `uuidv4()`, and then store the build ID reported by the test runner separately?

Reply:
Now stores both, with raw build id in `buildIdReal` and buildid + uuid in `buildIdReal`

Reply:
I think I'd prefer the "different" name be the one with the UUID. Something like `internalBuildId`, `buildIdInternal`, or `buildIdWithUuid`. I had to look a few times to understand/see `buildIdReal`.